docs/scos/user/back-office-user-guides/202108.0/merchandising/discount/creating-cart-rules.md

```diff
@@ -21,38 +21,36 @@ A cart rule is a discount that is applied automatically when all attached discou
 
 ## Prerequisistes
 
-To start working with discounts, go to **Merchandising** > **Discount**.
+To start working with discounts, go to **Merchandising&nbsp;<span aria-label="and then">></span> Discount**.
 
 ## Creating a cart rule discount
 
 To create a cart rule discount:
-1. On the *Discount* page, in the top right corner, click **Create new discount**.
-2. On the *Create Discount* page, in the *General information* tab, do the following:
+1. On the **Discount** page, in the top right corner, click **Create new discount**.
+2. On the **Create Discount** page, on the **General information** tab, do the following:
     1. In **Store relation**, check the stores you wish the discount to be active in.
     2. From the **Discount Type** drop-down list, select a cart rule.
     3. In the **Name** field, specify the name for the discount.
```

Comment:
We have an information blurb in the Back Office that describes the functions of discount name and description (shown in discount summary in the cart, shown in the promotional product widget in the cart), but I also wonder if we should include this here. It could be useful when we have full backend API functionality and the business user isn't necessarily engaging with our back office. What do you think?

Reply:
I think you are right. This might be useful and adds clarity on what to type in the **Name** and **Description** fields.